libjlm/include/jlm/backend/hls/rhls2firrtl/mlirgen.hpp

```diff
@@ -227,9 +227,11 @@ namespace jlm {
                         }
                         std::string get_text(jlm::RvsdgModule &rm) override {return "";}
                 public:
-                        void
+			std::string
                         run(jlm::RvsdgModule &rvsdgModule) {
                                 std::cout << "This version of jlm-hls has not been compiled with support for the CIRCT backend\n";
+				exit(EXIT_FAILURE);
+				return "";
```

Comment:
Identation!

Also, please throw an exception:

throw jlm::error("This version of jlm-hls has not been compiled with CIRCT support.");

Reply:
Changed